src/app/SharpRaven/IRavenClient.cs

```diff
@@ -215,7 +217,7 @@ public interface IRavenClient
 
 #endif
 
-        #region Deprecated Methods
+#region Deprecated Methods
```

Comment:
I think I like it better when all of the preprocessor directives (`#if`, `#region`, etc.) are aligned with the code they surround and not left-aligned. Can you please revert this and properly align all the `#if` processor directives you've added?